Sources/SwiftDocC/Semantics/Metadata/Metadata.swift

```diff
@@ -93,7 +102,7 @@ public final class Metadata: Semantic, AutomaticDirectiveConvertible {
     
     func validate(source: URL?, for bundle: DocumentationBundle, in context: DocumentationContext, problems: inout [Problem]) -> Bool {
```

Comment:
We should probably add a validation for the argument passed to the new `@PageColor` directive which emits a diagnostic when the user specifies a non-supported color name.

For example, the following doesn't work (as expected), but there is no warning by the compiler that `"teal"` is not one of the supported colors.

```markdown
# Example

@Metadata {
  @PageColor(teal)
}
...
```

Reply:
Oh strange... I expected that to already emit a warning. Did you try it?

Reply:
Yeah. I was using the preview command though, so maybe that might have made some difference? Not sure.

Reply:
Just ran into this as well. We're you using `bin/preview-docs` script in the repo? It looks like we pass `--diagnostic-level error` there for some reason which filtered it out.

When I remove that I see:
```
/DocCDocumentation.docc/customizing-the-appearance-of-your-documentation-pages.md:6:1: warning: Cannot convert 'teal' to type 'Color'
```

which seems correct.

Reply:
Ah, whoops. Yeah, that's what I did.